### PR TITLE
[FR] osu!_UCI

### PR DESCRIPTION
**Pull request by DoryanR**

### CODE REVIEW

File: wiki/Organisations/osu!_UCI/fr.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+- Collège
+- Irvine
+- Université
+---
+
+# osu! UCI
+
+**osu! UCI** est un club osu! centré sur le jeu et la discussion à l’Université de Californie, Irvine (*UCI*). Ils organisent divers événements, notamment des tournois LAN saisonniers et des rencontres hebdomadaires.
+
+## Links
```
Comment (almaloste) [inline wiki/Organisations/osu!_UCI/fr.md]:
```suggestion
## Liens
```

File: wiki/Organisations/osu!_UCI/fr.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+- **[Site Internet](http://www.osuuci.com)**
+- [Serveur Discord](https://discord.gg/qbZddFV)
+- [Groupe Facebook](https://www.facebook.com/groups/osuuci/)
+- [Chaîne Twitch](https://www.twitch.tv/osuuci)
+- [Twitter](https://twitter.com/osuUCI)
+
+## Les officiers
```
Comment (Pachiru) [inline wiki/Organisations/osu!_UCI/fr.md]:
```suggestion
## Staff
```

File: wiki/Organisations/osu!_UCI/fr.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+- Université
+---
+
+# osu! UCI
+
+**osu! UCI** est un club osu! centré sur le jeu et la discussion à l’Université de Californie, Irvine (*UCI*). Ils organisent divers événements, notamment des tournois LAN saisonniers et des rencontres hebdomadaires.
+
+## Liens
+
+- **[Site Internet](http://www.osuuci.com)**
```
Comment (cl8n) [inline wiki/Organisations/osu!_UCI/fr.md]:
```suggestion
- **[Site internet](http://www.osuuci.com)**
```
... i think

File: wiki/Organisations/osu!_UCI/fr.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+- Californie 
+- Club
+- Collège
+- Irvine
+- Université
+---
+
+# osu! UCI
+
+**osu! UCI** est un club osu! centré sur le jeu et la discussion à l’Université de Californie, Irvine (*UCI*). Ils organisent divers événements, notamment des tournois LAN saisonniers et des rencontres hebdomadaires.
```
Comment (cl8n) [inline wiki/Organisations/osu!_UCI/fr.md]:
```suggestion
**osu! UCI** est un club osu! centré sur le jeu et la discussion à l’Université de Californie à Irvine (*UCI*). Ils organisent divers événements, notamment des tournois LAN saisonniers et des rencontres hebdomadaires.
```
according to google + wikipedia, at least

**Pull request merged**